src/NBitcoin/Script.cs

```diff
@@ -807,8 +814,10 @@ private static BitcoinStream CreateHashWriter(HashVersion version)
 
         public IEnumerable<Op> ToOps()
         {
-            ScriptReader reader = new ScriptReader(_Script);
-            return reader.ToEnumerable();
+            using (ScriptReader reader = new ScriptReader(_Script))
+            {
+                return reader.ToEnumerable().ToList();
```

Comment:
Does `return reader.ToEnumerable();` not work here?

Reply:
No. It does git disposed before enumeration starts. 

But we can afford loading it as a whole list- scripts are small anyway and in every case that enumerable would enumerate it to the end so all of it will eventually be loaded.

Reply:
ah yes fair enough.